test/powershell/Language/Parser/UsingNamespace.Tests.ps1

```diff
@@ -10,27 +10,7 @@ using namespace System.Diagnostics; using namespace System.Runtime.CompilerServi
 using namespace System.Collections.Generic
 
 Import-Module $PSScriptRoot\..\LanguageTestSupport.psm1
-
-function ShouldBeErrorId
-{
-    param([Parameter(ValueFromPipeline, Mandatory)]
-          [ScriptBlock]
-          $sb,
-          
-          [Parameter(Mandatory, Position=0)]
-          [string]
-          $FullyQualifiedErrorId)
-
-    try
-    {
-        & $sb
-        throw "Unexpected"
-    }
-    catch
-    {
-        $_.FullyQualifiedErrorId | Should Be $FullyQualifiedErrorId
-    }
-}
+Import-Module -Force $PSScriptRoot\..\..\Common\Test.Helpers.psm1

```

Comment:
I will removed the Force then

---
In reply to: [79257879](https://github.com/Powershell/Powershell/pull/2218#discussion_r79257879) [](ancestors = 79257879,79249686)